provider/datagen/data/segmenter/grapheme.toml

```diff
@@ -49,16 +49,45 @@ name = "LVT"
 [[tables]]
 name = "Extended_Pictographic"
 
+[[tables]]
+name = "InCBLinker"
+
+[[tables]]
+name = "InCBConsonant"
+
+[[tables]]
+name = "InCBExtend"
+
 [[tables]]
 name = "Extended_Pictographic_Extend"
 left = "Extended_Pictographic"
 right = "Extend"
 
+[[tables]]
+name = "Extended_Pictographic_Extend"
+left = "Extended_Pictographic"
+right = "InCBLinker"
+
+[[tables]]
+name = "Extended_Pictographic_Extend"
+left = "Extended_Pictographic"
+right = "InCBExtend"
```

Comment:
@eggrobin looks like this accounts for InCBLinker/InCBExtend characters that are also Extend and thus need to be handled by [GB11](https://www.unicode.org/reports/tr29/#GB11).

Is there any likelihood InCB will change such that they contain things that are _not_ Extend?

we should probably have comments documenting this here anyway?